python/cli/prelude_cli/views/build.py

```diff
@@ -42,22 +42,41 @@ def create_test(controller, name, unit, test, techniques, advisory):
         )
 
     if not test:
+        """ Mark-up, upload, and write the test file template. """
         basename = f'{t["id"]}.go'
-        template = pkg_resources.read_text(templates, 'template.go')
-        template = template.replace('$ID', t['id'])
-        template = template.replace('$NAME', name)
-        template = template.replace('$UNIT', unit or '')
-        template = template.replace('$CREATED', str(datetime.utcnow()))
+        utcTime = str(datetime.utcnow())
+        testTemplate = pkg_resources.read_text(templates, 'template.go')
```

Comment:
Can you change the var names to be more consistent with how the codebase handles python vars. Words separated by underscores.

```suggestion
        test_template = pkg_resources.read_text(templates, 'template.go')
```